debian/control

```diff
@@ -31,6 +31,7 @@ Build-Depends: debhelper (>= 9.20141010),
                libselinux1-dev (>= 2.1.9),
                libacl1-dev,
                liblzma-dev,
+               liblz4-dev (>= 0.0~r125),
```

Comment:
that said looking into the source it's really only `test-compress` that calls it and it shouldn't be called without lz4.

it was introduced in debian in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=832010 I guess we can remove it.. let me retry a build without it.